### PR TITLE
Add a new CDEPS test to use GSFv16 data mode in CDEPS ,update rt_weekly.conf and update MOM6 to main branch (20211220)

### LINKED ISSUE #949: Add a new cdeps test to validate GFS v16 data mode in CDEPS
## Description
The GFS v16 data mode will be added to the develop branch of NOAA-EMC/CDEPS next week from PR#40. A new test (“datm_cdeps_3072x1536_cfsr”) needs to be added to validate the new data mode.

## Solution
The new test will be created with the following steps:
1) A new ESMF mesh file will be created using GFS v16 forcing grid (Gaussian grid: 3072x1536).
2) The current CFSR forcing files in the RT input directory will be interpolated to the GFS v16 grid using ESMF regridding tools.
3) A new file for the input data will be added to the fv3_conf directory.
4) The new test will be added to the tests directory.

Both the new test and datm_cdeps_control_cfsr test will be run for 5 days and results from these two runs will be analyzed and compared.

Once GFS v16 forcing files are available, the datm_cdeps_3072x1536_cfsr will be converted to datm_cdeps_gfsv16 by replacing the input forcing files with GFS v16 data.

## Alternatives
If applicable, add a description of any alternative solutions or features you've considered.

## Related to
- https://github.com/NOAA-EMC/CDEPS/pull/40

### LINKED ISSUE #956: Update regression test scripts for weekly tests
## Description
The current regression test scripts are designed to either create a new baseline or compare output files with those from previous baseline automatically after the models are completed.

For the weekly tests, we only want to know if the models are completed successfully. To do so, we need to update several scripts.

## Solution
The following scripts will be updated:
rt.sh
run_test.sh
rt_utils.sh

### PR DESCRIPTION
**Pull request by binli2337**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [x] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [x] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

A new CDEPS test "datm_cdeps_3072x1536_cfsr" will be added. This new test will use GFSv16 data mode. The CFSR forcing files are obtained by interpolating the original CFSR data to the GFSv16 grid.

The rt_weekly.conf will also be updated to include "cpld_bmark_p7_35d" tests.

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #949 
- fixes #956 

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [skip - maintenance] cheyenne.intel 
- [skip - maintenance] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [ ] wcoss_cray
- [x] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [x] CI

## Dependencies
No dependencies

Co-authored-by: jiandewang

### CODE REVIEW

File: tests/tests/cpld_bmark_p7_35d
Comment (MinsukJi-NOAA) [inline tests/tests/cpld_bmark_p7_35d:20]: Did you mean to change to DAYS=1 and FHMAX=24? If commented out, what values are used?

**Comment by climbfuji** [comment]
@binli2337 Do we need new input data? If so, please let me know what to transfer to Cheyenne (if I can still access it during the maintenance period). Also, please fill out the PR template above.

File: tests/tests/cpld_bmark_p7_35d
Comment (binli2337) [reply tests/tests/cpld_bmark_p7_35d:20]: DAYS=1

**Comment by junwang-noaa** [comment]
@binli2337 Do your code updates change current baseline results? If not, maybe you can just add the new test baseline to current baseline?

**Comment by binli2337** [comment]
@climbfuji The new data directory is /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/input-data-20211210/DATM_CDEPS/CFSR3072x1536. Please transfer the new data directory to Cheyenne if possible. Thanks!

**Comment by binli2337** [comment]
@junwang-noaa The updates will not change current results. I will add a new test directory to develop-20211222.

**Comment by climbfuji** [comment]
> /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/input-data-20211210/DATM_CDEPS/CFSR3072x1536

The new input data is now on Cheyenne.

**Comment by jiandewang** [comment]
@binli2337 can you modify the PR title to include "update MOM to 20211220" ?

**Review by junwang-noaa — APPROVED** [review]

**Comment by jiandewang** [comment]
@binli2337 I will merge MOM6 once it's approved by reviewer

**Review by jiandewang — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]

**Comment by jiandewang** [comment]
merged MOM6 to dev/emc. @binli2337 can you change the MOM6 submodule ?

**Comment by climbfuji** [comment]
Submodule pointer for MOM6 is correct (32c0e1e) - ready to merge?

**Review by MinsukJi-NOAA — APPROVED** [review]

**Comment by climbfuji** [comment]
I think @pjpegion is on leave, so let's go ahead and merge it - everyone else listed as reviewer approved it.

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: cheyenne
Compiler: gnu
Job: RT
Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/809293892/20211229061511/ufs-weather-model
Please manually delete: /glade/scratch/dtcufsrt/FV3_RT/rt_612
Cannot upload cheyenne.gnu RT LogIt is blocked by PR owner
Please obtain logs from /glade/scratch/dtcufsrt/autort/tests/auto/pr/809293892/20211229061511/ufs-weather-model

**Pull request merged**
